pkg/storegateway/bucket_stores.go

```diff
@@ -293,7 +293,7 @@ func (u *BucketStores) syncUsersBlocks(ctx context.Context, f func(context.Conte
 	return errs.Err()
 }
 
-// Series makes a series request to the underlying user bucket store.
+// Series implements the StoreGateway proto service, making a series request to the underlying user bucket store.
```

Comment:
It implements StoreServer interface. We can be explicit, and let type linking in comments do the job:

```suggestion
// Series implements storepb.StoreServer interface, making a series request to the underlying user bucket store.
```